website/blog/2022-08-08-benchmark-tpc-h.md

```diff
@@ -156,7 +156,7 @@ The code below connects to Databend using the Root user. Please note that the ro
 for t in customer lineitem nation orders partsupp part region supplier
 do
     echo "$t"
-    curl -XPUT 'http://root:@127.0.0.1:8000/v1/streaming_load' -H 'insert_sql: insert into tpch.'$t' format CSV' -H 'skip_header: 0' -H 'field_delimiter:|' -H 'record_delimiter: \n' -F 'upload=@"./'$t'.tbl"'
```

Comment:
why `'skip_header: 0'` and `'record_delimiter: \n'` are deleted?

Reply:
these are default values

Reply:
add back `-H 'record_delimiter: \n' ` since the default is `\r\n`